python/src/ir.cc

```diff
@@ -721,12 +721,14 @@ void init_triton_ir(py::module &&m) {
              return self.getBuilder().getI64Type();
            })
       .def("get_fp8e4nv_ty",
+            // TODO: fp8e4nv is using Float8E4M3FNUZType, which 
+            // does not seem right.
            [](TritonOpBuilder &self) -> Type {
              return self.getBuilder().getType<Float8E4M3FNUZType>();
            })
       .def("get_fp8e4b8_ty",
-           [](TritonOpBuilder &self) -> mlir::Type {
-             return self.getBuilder().getType<mlir::Float8E4M3FNUZType>();
+           [](TritonOpBuilder &self) -> Type {
+             return self.getBuilder().getType<Float8E4M3FNUZType>();
```

Comment:
interesting, so we picked the wrong type for fp8e4 format here? I think you are right and for nv we should use `Float8E4M3FN`. Can you mention that in the TODO?

Reply:
Done.